### PR TITLE
build: bump version keri to 1.1.16

### PR DESCRIPTION
**Pull request by minhdo6491**
## Context:
- Bump the version for the keri

## Change
- Bump version to 1.1.16 (verified version)
- Modify version wait

## Ticket:
- https://cardanofoundation.atlassian.net/browse/DTIS-982

### CODE REVIEW

File: src/backer/backering.py [hunk 1]
```diff
@@ -221,28 +230,6 @@ def cueDo(self, tymth=None, tock=0.0):
...
-        """
-        self.wind(tymth)
-        self.tock = tock
-        _ = (yield self.tock)
-
-        while True:
-            for rep in self.exc.processResponseIter():
-                self.replies.append(rep)
-                yield  # throttle just do one cue at a time
-            yield
```
Comment (iFergal) [inline src/backer/backering.py:245]: Hmm, why is this removed, is it actually not needed for the backer to work?

Comment (minhdo6491) [reply src/backer/backering.py:245]: the processResponseIter outdated, and remove from the Exchanger
https://github.com/WebOfTrust/keripy/blob/3febc54a463125a4cc3d86d694e57004134b2092/src/keri/peer/exchanging.py#L22

Comment (minhdo6491) [reply src/backer/backering.py:245]: Also take a look at keripy, keria, it seems that exchangeDo, processResponseIter are no longer in used

**Comment by minhdo6491** [comment]
Here is the verifcation, when e2e, signts -> keria -> cardano-backer

image

Will add some UTs for the verification

File: src/backer/backering.py (hunk 1 above)
Comment (iFergal) [reply src/backer/backering.py:245]: got it, checked the history and this was removed during a major refactor of exchange messages, all good

**Review by iFergal — APPROVED** [review]
